Octokit/Models/Request/NewDeployment.cs

```diff
@@ -59,7 +60,7 @@ public NewDeployment(string @ref)
         /// <summary>
         /// JSON payload with extra information about the deployment.
         /// </summary>
-        public string Payload { get; set; }
+        public Dictionary<string, string> Payload { get; set; }
```

Comment:
I'm curious whether this can be any sort of JSON object, and if we're being too strict here with assuming values will always be `string`. I can't find any interesting examples in the docs beyond [this mention](https://docs.github.com/en/rest/reference/repos#create-a-deployment) but maybe you're more familiar with this API currently than I am:

> The payload parameter is available for any extra information that a deployment system might need. **It is a JSON text field that will be passed on when a deployment event is dispatched.**